src/main/java/io/cdap/plugin/gcp/bigquery/sink/lib/BigQueryOutputConfiguration.java

```diff
@@ -0,0 +1,525 @@
+/*
+ * Copyright 2017 Google LLC
```

Comment:
Update docs for each new class as per apache copyright.

Reply:
```java
/*
 * Copyright © 2023 Cask Data, Inc.
 *
 * Licensed under the Apache License, Version 2.0 (the "License"); you may not
 * use this file except in compliance with the License. You may obtain a copy of
 * the License at
 *
 * http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
 * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 * License for the specific language governing permissions and limitations under
 * the License.
 */
```